src/Common/nunit/ConsoleOptions.cs

```diff
@@ -382,8 +382,10 @@ private void ConfigureOptions()
             this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
                 v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
 
+#if !NETCF
```

Comment:
So perhaps:

 

#if !NUNITLITE || !NETCF

 

?

 

From: Vladimir Kloz [mailto:notifications@github.com] 
Sent: Monday, April 27, 2015 5:55 AM
To: nunit/nunit
Cc: Neil Colvin
Subject: Re: [nunit] Issue #611: Remove unused --teamcity option from CF build of NUnitLite (#619)

 

In src/Common/nunit/ConsoleOptions.cs <https://github.com/nunit/nunit/pull/619#discussion_r29114264> :

> @@ -382,8 +382,10 @@ private void ConfigureOptions()
>              this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
>                  v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
>  
> +#if !NETCF

It will need IMHO more thourough condition then since the code seems to be reused by other parts as well. So this way the option will be supported only in NUNITLITE which is most likely incorrect too.

—
Reply to this email directly or view it on GitHub <https://github.com/nunit/nunit/pull/619/files#r29114264> .  <https://github.com/notifications/beacon/AFh64UoY8vkjCIGL9MmlatTePf1D9xwnks5oDVZKgaJpZM4EJMBn.gif>